rollingpin/args.py

```diff
@@ -258,11 +276,11 @@ def construct_canonical_commandline(config, args):
         arg_list.extend(args.components)
 
     for command in args.commands:
-        if command[0] == "restart":
-            arg_list.extend(("-r", command[1]))
+        if isinstance(command, commands.RestartCommand):
```

Comment:
We can probably invert this logic somehow to avoid `isinstance` checks now but I didn't want to go super hard on the refactor and complicate the PR